### PR TITLE
🪟 🎉 Free Connectors Program large enrollment callout

### LINKED ISSUE #21223: [Free connectors] create "large" enrollment banner
Part of https://github.com/airbytehq/airbyte/issues/21074. For airbyte employees, a Figma mockup can be found [here](https://www.figma.com/file/wihAJg5mkp61I7lLHs0IpV/FREE-CONNECTOR-PROGRAM?node-id=1095%3A15686&t=G4oMKBThKe3rYe6j-0); it looks like this
2023-01-10-11:18:58-screenshot

As with https://github.com/airbytehq/airbyte/issues/21222, clicking on this will trigger displaying the enrollment modal; until that exists, an empty placeholder click handler is fine.

### PR DESCRIPTION
**Pull request by teallarson**
## What

Screenshot 2023-01-13 at 2 54 45 PM

closes #21223

## How
Adds large banner and necessary assets

## Recommended reading order
1. `LargeEnrollmentBanner.*`
2. others

### CODE REVIEW

File: airbyte-webapp/src/packages/cloud/components/experiments/FreeConnectorProgram/longtail-connectors.svg [hunk 1]
```diff
@@ -0,0 +1,667 @@
+<svg width="165" height="120" viewBox="0 0 165 120" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (tealjulia) [inline airbyte-webapp/src/packages/cloud/components/experiments/FreeConnectorProgram/longtail-connectors.svg:1]: It looks like this may be the same image @josephkmh used in https://github.com/airbytehq/airbyte/pull/21396

Will adjust to use his instead once it's merged!

**Review by joeykmh — COMMENTED** [review]
Just left a few comments for the time being, will do a full review when #21315 is merged

File: airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/CreditsPage.tsx [hunk 2]
```diff
@@ -26,6 +27,7 @@ const CreditsPage: React.FC = () => {
       <div className={styles.content}>
         {!emailVerified && <EmailVerificationHint className={styles.emailVerificationHint} />}
         <RemainingCredits selfServiceCheckoutEnabled={emailVerified} />
+        <LargeEnrollmentCallout />
```
[the remark this answers is not in the document]
Comment (tealjulia) [reply airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/CreditsPage.tsx]: yep!  was waiting on the api info.  added in  https://github.com/airbytehq/airbyte/pull/21369/commits/632006951131de2811a2a3db8bf4c5b0efe0c069

File: airbyte-webapp/src/packages/cloud/components/experiments/FreeConnectorProgram/LargeEnrollmentCallout.module.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+.flexRow {
+  width: 100%;
+}
+
+.enrollButton {
+  margin-left: auto;
+}
+
+.textContainer {
+  p,
```
[the remark this answers is not in the document]
Comment (tealjulia) [reply airbyte-webapp/src/packages/cloud/components/experiments/FreeConnectorProgram/LargeEnrollmentCallout.module.scss]: addressed in https://github.com/airbytehq/airbyte/pull/21369/commits/7995c757f5214c6c193d2b7faa044d14d5d767bd

File: airbyte-webapp/src/packages/cloud/components/experiments/FreeConnectorProgram/LargeEnrollmentCallout.module.scss [hunk 4]
```diff
@@ -0,0 +1,31 @@
+@use "scss/variables";
+@use "scss/colors";
+
+.container {
+  margin: variables.$spacing-lg 0;
+  padding: variables.$spacing-lg variables.$spacing-xl;
```
[the remark this answers is not in the document]
Comment (tealjulia) [reply airbyte-webapp/src/packages/cloud/components/experiments/FreeConnectorProgram/LargeEnrollmentCallout.module.scss:6]: addressed in https://github.com/airbytehq/airbyte/pull/21369/commits/7995c757f5214c6c193d2b7faa044d14d5d767bd

File: airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/components/RemainingCredits.tsx [hunk 5]
```diff
@@ -28,7 +28,7 @@ interface Props {
 const Block = styled.div`
   background: ${({ theme }) => theme.blue50};
   border-radius: 8px;
-  padding: 18px 25px 22px;
+  padding: 15px 20px;
   font-size: 13px;
   line-height: 20px;
   display: flex;
```
Comment (tealjulia) [inline airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/components/RemainingCredits.tsx:34]: Worth converting this away from styled-components, use our actual variables, etc. ... but in the interest of getting this merged this AM that will be a separate PR!

File: airbyte-webapp/src/packages/cloud/components/experiments/FreeConnectorProgram/longtail-connectors.svg (hunk 1 above)
Comment (tealjulia) [reply airbyte-webapp/src/packages/cloud/components/experiments/FreeConnectorProgram/longtail-connectors.svg:1]: Actually, this is a different image!

File: airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/CreditsPage.tsx [hunk 6]
```diff
@@ -18,6 +21,10 @@ import styles from "./CreditsPage.module.scss";
 const CreditsPage: React.FC = () => {
   const { emailVerified } = useAuthService();
   useTrackPage(PageTrackingCodes.CREDITS);
+  const { data } = useFreeConnectorProgramInfo();
+  const isFreeConnectorProgramEnabled = useExperiment("workspace.freeConnectorsProgram.visible", false);
+  const showEnrollmentUi = isFreeConnectorProgramEnabled && data?.showEnrollmentUi;
```
Comment (josephkmh) [inline airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/CreditsPage.tsx:26]: As mentioned on slack, I wonder if we can just roll the `useExperiment()` call into `useFreeConnectorProgramInfo()` so that `data.showEnrollmentUi = false` if the experiment is disabled.

**Review by joeykmh — APPROVED** [review]
Changes look good to me! One non-blocking comment

**Review by ambirdsall — APPROVED** [review]
LGTM. Some slight overlap with the email verification PR, but that's no biggie.
